extensions/raycast-icons/src/components/IconGrid.tsx

```diff
@@ -43,8 +52,10 @@ export function IconGrid() {
     <Grid
       columns={8}
       inset={Grid.Inset.Large}
-      isLoading={false}
+      isLoading={isAILoading}
+      navigationTitle={selectedIconName ? `Search Raycast Icons – ${selectedIconName}` : "Search Raycast Icons"}
```

Comment:
We can't change the `navigationTitle` like that; it should not be altered in the root of a command. Therefore, we can either proceed as before with the truncated version or add it to the actions themselves  

What do you think, @xilopaint 😊

Reply:
@pernielsentikaer I'm not sure I get what you're saying, but the built-in emoji picker does the same thing with the `navigationTitle`:

<img width="886" alt="Screenshot 2025-05-15 at 07 49 20" src="https://github.com/user-attachments/assets/4f45552e-9459-42d1-8928-2d6f490a2f06" />

Reply:
Let's allow it; however, the above is not an extension and has a different ruleset :)